test/basic/vpc_create/vpc_create_test.go

```diff
@@ -101,4 +101,8 @@ func TestVPCCreate(t *testing.T) {
 	assert.Nil(t, networkAnalysisErr)
 	assert.Equal(t, len(networkAnalysis.NetworkInsightsAnalyses), 1)
 	assert.Equal(t, *networkAnalysis.NetworkInsightsAnalyses[0].Status, "succeeded")
+
+	tagged_resources, err := utils.GetTaggedResources(ctx, os.Getenv("ENVIRONMENT_ID"), vpcAsset.Id)
```

Comment:
casing consistency nitpick, we should def update our lint rules, something's missing on that, but i'm okay with this and we can revisit these in lump sum to be honest

Reply:
Thanks! My brain is so used to typing python these days that I keep doing this.